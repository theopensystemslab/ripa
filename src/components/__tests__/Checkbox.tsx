import { render } from "@testing-library/react";
import * as React from "react";

import StyledCheckbox from "../Checkbox";

describe("Styled Checkbox Component", () => {
  it("should render snapshot", async () => {
    const { asFragment } = render(<StyledCheckbox />);
    expect(asFragment()).toMatchSnapshot();
  });
});
