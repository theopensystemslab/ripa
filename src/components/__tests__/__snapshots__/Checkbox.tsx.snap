// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Checkbox Component should render snapshot 1`] = `
<DocumentFragment>
  <span
    aria-disabled="false"
    class="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-10 MuiCheckbox-root makeStyles-checkBoxRoot-1 MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
  >
    <span
      class="MuiIconButton-label"
    >
      <input
        class="PrivateSwitchBase-input-13"
        data-indeterminate="false"
        type="checkbox"
        value=""
      />
      <span
        class="makeStyles-icon-2"
        font-size="default"
      />
    </span>
    <span
      class="MuiTouchRipple-root"
    />
  </span>
</DocumentFragment>
`;
